SR2021/2021-01-08-code-submitter-credentials.md

```diff
@@ -0,0 +1,22 @@
+---
+to: SR2021 teams
+subject: Student Robotics Code Submitter Credentials
+---
+
+Hi,
+
+Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.
+
+The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.
+
+| TLA | Password |
```

Comment:
It's probably worth mentioning somewhere that their TLA is their username.

Reply:
aha, good catch!